gcloud-java-bigquery/src/main/java/com/google/gcloud/bigquery/ExtractJobConfiguration.java

```diff
@@ -18,23 +18,22 @@
 
 import static com.google.common.base.Preconditions.checkNotNull;
 
-import com.google.api.services.bigquery.model.Job;
-import com.google.api.services.bigquery.model.JobConfiguration;
 import com.google.api.services.bigquery.model.JobConfigurationExtract;
-import com.google.common.base.MoreObjects.ToStringHelper;
+import com.google.common.base.MoreObjects;
 import com.google.common.collect.ImmutableList;
-import com.google.gcloud.bigquery.JobStatistics.ExtractStatistics;
 
+import java.io.Serializable;
 import java.util.List;
 import java.util.Objects;
 
 /**
- * Google BigQuery Extract Jobs. An Extract Job exports a BigQuery table to Google Cloud Storage.
- * The extract destination provided as URIs that point to objects in Google Cloud Storage.
+ * Google BigQuery Extract Job configuration. An Extract Job exports a BigQuery table to Google
```

Comment:
Do you think Extract and Job should be capitalized in this case?
Should we mention that this configuration is associated with JobConfiguration.Type.EXTRACT?